src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import './Todo.css'
+import TodoList from './TodoList'
+import TodoForm from './TodoForm'
+
+
+
+const newTodo = [
+    {
+      task: 'Organize Garage',
+      id: 1528817077286,
+      completed: false
+    },
+    {
+      task: 'Bake Cookies',
+      id: 1528817084358,
+      completed: false
+    }
+  ];
+
+class Todo extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+
+            moreTodo: newTodo,
+            task: '',
+            id: Date.now(),
```

Comment:
You are initializing the `this.state.id` with `Date.now()` and never actually update it, so there's no need to put it in a state, you can use simply a variable for that. But I guess what you want to do here is to make unique id for each todo, if so, then you need to update this `id` each time you create a new todo.